### PR TITLE
Revamp readme

### PR DESCRIPTION
**Pull request by jessb0t**
@F-said:
1. I re-worked your content for improved readability and to align better with our new readme template.  Please check my work to ensure I did not leave anything off or confuse anything!
2. One question came up in the process, regarding this command in the information on how to build the wiki on local:
image
I'm wondering if we should put that content within the `main` _config.yml file so that no one would need to add it in order to build the wiki on local?  Or does that not work?  (This isn't a request, just a question of whether that would make it simpler for future users.  Thanks!)

@georgebuzzell:
Please also review this revision to the readme, which will hopefully clarify some of your questions on how to add images, pages, and new sections to the wiki. 🤓

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,217 +49,213 @@ wiki
...
-    plugins:
-        - jekyll-remote-theme
-    ```
+This method for linking images has two key benefits:
+1. The image file is easily accessible within the wiki repo.
+2. Given that the `main` branch is the published branch, the image displays correctly upon publication.
 
-10. Run the following command to build the website on your local machine. 
-    > bundle exec jekyll serve
+However, this method does have one major disadvantage of which users should be aware: the image will **not** render correctly on their working branch until it is merged with `main`. As a result, users are encouraged to double-check all images after the merge with `main` to ensure they are displaying as intended.
```
Comment (georgebuzzell) [inline README.md:86]: Is there any way around this to allow for users to be able to properly view the embedded images before the branch is merged into main? It seems like this is a fairly big issue, no?

Comment (georgebuzzell) [reply README.md:86]: @F-said @jessb0t

File: README.md [hunk 2]
```diff
@@ -31,217 +49,213 @@ wiki
...
 
-### Page Creation 
+## Adding Pages
 
-To create a page on the wiki and link it to the left-hand navigation bar, create the markdown file inside of the [docs/](https://github.com/NDCLab/wiki/tree/gh-pages/docs) directory and specify its' position on the navbar in the header.  
+To add a brand new page on the wiki, create a new markdown file inside of the appropriate directory. This markdown file should begin with the following content, which specifies its name and nav order in the wiki structure:
```
Comment (georgebuzzell) [inline README.md:90]: i dont think markdown was defined (and/or linked to a definition and cheat sheet) on this page.

Comment (georgebuzzell) [reply README.md:90]: @F-said @jessb0t

File: README.md [hunk 3]
```diff
@@ -31,217 +49,213 @@ wiki
...
 
-```yml
----
-layout: default
-title: Onboarding
-nav_order: 2
-has_children: true
-permalink: /docs/onboarding
----
+To view what the site would look like with the applied formatting and styles as seen on the live site, you can build a local copy by following the steps listed below.
```
Comment (georgebuzzell) [inline README.md:161]: OK, so this is the workaround to checking how the rendered images will look, right? If so, we should mention this above as well @F-said @jessb0t

**Review by georgebuzzell — APPROVED** [review]
I left minor comments that should be addressed. However, since these are minor, I am going to go ahead and approve. Once these comments are addressed, please merge and delete this branch.

FYI: I do continue to wonder if we need a paralell, easier to manipulate wiki, but, i think continuing to move forward with what we have is great. Ultimately, what I am considering is parallel methods of modifying the wiki via the gh/markdown approach or the doku wysiwyg approach (that can be integrated fully with gh ). There appear to be ways to set up a wiki such that it can allow for either method. But, this is not something to worry about now at all. I only mention it so it is not a surprise if I bring it up in the future. And, we would not be replacing the amazing work that you all have done, just expanding the usability of it (potentially).

**Comment by georgebuzzell** [comment]
@F-said @jessb0t Approved with minor modifications needed. Once these are addressed, please go ahead and merge/delete.

File: README.md (hunk 1 above)
Comment (jessb0t) [reply README.md:86]: The only workarounds I have found are:

1. Code the path with the dev branch first, then switch that at PR.  (Which anyone can do if they want, of course.)
2. Use a relative path, which I just tested.  It renders properly on GitHub Preview...but is not rendered properly by GitHub pages.

But, to be honest, this hasn't felt like a major impediment to me and everyone who is working through Get with Git basically sees it on their very first GitHub activity in the lab.

Comment (jessb0t) [reply README.md:86]: P.S. I did a bunch of hunting around GitHub and Stack Overflow, so this isn't based solely on my own testing.

File: README.md (hunk 2 above)
Comment (jessb0t) [reply README.md:90]: @georgebuzzell Do you feel that this is needed, given that **all** readmes on GitHub use markdown by necessity?  Note, also, that Get with Git explicitly contains a whole module on Markdown, offered directly from GitHub.

Happy to add, I just want to make sure we don't clutter with too many basics and I think knowledge of Markdown will be something that people already have before they poke around this readme.  But I could be wrong!

File: README.md (hunk 3 above)
Comment (jessb0t) [reply README.md:161]: Actually, no...for this to work, one would have to hard-code the dev branch into the image link, then change to "main" before PR.  I think that it is fundamentally a discrepancy between Markdown and HTML (if I understood some of the Stack Overflow comments correctly).

File: README.md (hunk 2 above)
Comment (georgebuzzell) [reply README.md:90]: I think I am just looking to a link to the page where it is defined is all

@jessb0t

**Comment by georgebuzzell** [comment]
@jessb0t I am still a little unclear on the issue with previewing the rendering of images. But, also sounds like not something to hold things up over or prioritize workarounds (and maybe there are none).

All in all, please just link to markdown and then I think this is good to go 

@jessb0t @F-said

File: README.md (hunk 2 above)
Comment (jessb0t) [reply README.md:90]: It is here!
https://ndclab.github.io/wiki/docs/technical-docs/git_and_github.html#markdown

**Comment by F-said** [comment]
@jessb0t 

just made the following changes:

- placed ```plugins``` into _config
- removed _config step
- standardized windows/linux & mac steps 

You're right, would be easier to simply change the _config file as opposed to guiding users.

@georgebuzzell 
I've been previewing images on the wiki by uploading the images to github and then pasting the link inside of our markdown. 

Namely, I could simply drag an image from my PC or the web  into this textbox and it would manually upload it and provide a link I could use:

NDCLabWebLogo

**Review by F-said — APPROVED** [review]

**Comment by jessb0t** [comment]
Thanks, @F-said !  Yeah, it's true that's a workaround.  One thing I don't like, though, is that the base image isn't part of _assets when we do this.  I'll put this on our devOps discussion board, though, so we can talk it out and figure out a standard for the future.  Really good point that this solves it...

Merging now!

File: README.md (hunk 3 above)
Comment (jessb0t) [reply README.md:161]: Added to devOps agenda.

**Pull request merged**